components/awards/ScoreBasedAwards.js

```diff
@@ -1,30 +1,72 @@
 import React, { useState, useEffect } from 'react'; // React
+import { useRouter } from 'next/router'; // Collect Router
+import axios from 'axios'; // Axios
 import Layout from '@components/Layout'; // Layout wrapper
 
+import Navigation from '@containers/Navigation'; // Navigation state
+import Loader from 'react-loader-spinner'; // Spinning loader
 import Title from '@components/Title'; // Title
 import Input from '@components/Input'; // Input
 import Tabs from '@components/Tabs'; // Tabs
 import Table from '@components/Table'; // Table
+import Modal from '@components/Modal.js'; // Modal
 import Pagination from '@components/Pagination'; // Pagination
 import Search from '@assets/search.svg'; // Search icon
 import DancerRedJump from '@assets/dancer-red-jump.svg'; // Jumping Dancer SVG
 import DancerYellowBlue from '@assets/dancer-yellow-blue.svg'; // Jumping Dancer SVG
-import styles from '@styles/pages/Performances.module.scss'; // Page styles
-
-// Temp constants
-import events, { columns } from '../../data/mockParticipants';
+import styles from '@styles/pages/Awards.module.scss'; // Page styles
+import { formatPerformances } from '@utils/performances'; // Format performances util
+import AwardPill from '@components/awards/FinalizePill.js'; // Award Finalize Pill
 
 const PAGE_SIZE = 8; // Rows per page
```

Comment:
For performance page I think we're showing 20 rows per page.  Maybe we can change this to 20 as well here?